crypto/provider_core.c

```diff
@@ -1121,11 +1126,9 @@ static int core_thread_start(const OSSL_CORE_HANDLE *handle,
  */
 #ifndef FIPS_MODULE
 /*
- * TODO(3.0) These error functions should use |handle| to select the proper
- * library context to report in the correct error stack, at least if error
+ * These error functions should use |handle| to select the proper
+ * library context to report in the correct error stack if error
  * stacks become tied to the library context.
- * We cannot currently do that since there's no support for it in the
- * ERR subsystem.
```

Comment:
Why did you remove this part of the comment?  I see no reason to.

Reply:
I thought it is kind of implied from the sentence before, but I'll add that back.